Web/Components/Pages/Login.razor.css

```
@@ -0,0 +1,74 @@
+    margin-bottom: 1rem;
+}
+
+::deep .subtitle {
+    font-size: 1.15rem;
+    color: #e0e0e0;
+    margin-bottom: 1rem;
+    text-align: center;
+    font-family: 'Segoe UI', Arial, sans-serif;
+}
+
+::deep .sign-in-btn {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)